server/datastore/mysql/software_installers.go

```diff
@@ -405,7 +405,7 @@ WHERE
 	return &dest, nil
 }
 
-var errDeleteInstallerWithAssociatedPolicy = errors.New("Couldn't delete. Policy automation uses this software. Please disable policy automation for this software and try again.")
+var errDeleteInstallerWithAssociatedPolicy = &fleet.BadRequestError{Message: "Couldn't delete. Policy automation uses this software. Please disable policy automation for this software and try again."}
```

Comment:
Can we use a ConflictError (409) here instead?

Reply:
Sure why not!